tradewave/templates/tradewave/header.html

```diff
@@ -1,47 +1,51 @@
 <!-- Header Wrapper -->
-    <div id="header-wrapper" class="wrapper">
-        <div class="container">
-            <div class="row">
-                <div class="12u">
+<div id="header-wrapper" class="wrapper">
+    <div class="container">
+        <div class="row">
+            <div class="12u">
 
-                    <!-- Header -->
-                        <div id="header">
+                <!-- Header -->
+                <div id="header">
 
-                            <!-- Logo -->
-                                <div id="logo">
-                                    <h1><a href="#">Trade Wave</a></h1>
-                                    <span class="byline">Creating a Sustainable Economy</span>
-                                </div>
-                            <!-- /Logo -->
+                    <!-- Logo -->
+                    <div id="logo">
+                        <h1><a href="#">Trade Wave</a></h1>
+                        <span class="byline">Creating a Sustainable Economy</span>
+                        {% if status_msg %}
```

Comment:
@Urielar `status_msg` will contain a string every time we want to show a message to the user. I plugged it in here for now. Can you please check if there is a better place for it on the page and make it look nicer?